src/CMakeLists.txt

```diff
@@ -325,29 +325,28 @@ if((APPLE) AND (PKG_CONFIG_FOUND))
   endif()
 endif()
 
-find_package(Qt5Core REQUIRED)
+find_package(Qt5 5.6 REQUIRED COMPONENTS Core Multimedia Network OpenGL UiTools Widgets Concurrent)
+
+# This is not present before Qt 5.8:
+find_package(Qt5 COMPONENTS Gamepad QUIET)
+
 if (Qt5Core_VERSION VERSION_LESS 5.6)
   message(FATAL_ERROR "Mudlet requires Qt 5.6 or later")
 else()
-  message(STATUS "Qt version: ${Qt5Core_VERSION}")
+  if (Qt5Core_VERSION VERSION_LESS 5.8)
+    message(WARNING "Qt version: ${Qt5Core_VERSION} - no Gamepad support, it requires 5.8 at a minimum")
```

Comment:
There was already a message about enabled gamepad support in line 413. Do we really want to have that doubled up?

Reply:
As it happens this message is only going to appear on old Qt versions which can't support it - whereas the message on line 413 about using the QtGamepad module can only appear when it is...